x-pack/plugins/case/server/routes/api/types.ts

```diff
@@ -32,12 +32,14 @@ export interface CaseAttributes extends NewCaseType, SavedObjectAttributes {
 
 export type FlattenedCaseSavedObject = CaseAttributes & {
   case_id: string;
+  version?: string;
```

Comment:
we should always have a version, is that correct?

Reply:
its `?` in the `SavedObject` type: https://github.com/elastic/kibana/blob/master/src/core/server/saved_objects/types.ts#L73

Reply:
I think that we can default it to zero in our end so we do not have to deal with the undefined
type